package/com.unity.formats.usd/Samples/ImportMaterials/Editor/ImportMaterialsExampleEditor.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Unity Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Formats.USD.Examples
+{
+    [CustomEditor(typeof(ImportMaterialsExample))]
+    public class ImportMaterialsExampleEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            DrawDefaultInspector();
+
+            ImportMaterialsExample script = (ImportMaterialsExample)target;
+
+            if (GUILayout.Button("1. Initialize USD Package"))
+            {
+                script.InitializeUsd();
+            }
+
+            if (GUILayout.Button("2. Initialize Sample Shader Map Dictionary"))
+            {
+                script.InitializeSampleShaderMapDictionary();
```

Comment:
I don't think it makes sense to do each of these steps separately. They aren't descriptive of what they're doing or why they're needed, and if a user does them out of order it results in errors. I think it would be reasonable to put them all into one step of 'Import USD with Materials', and a second step to enter playmode to show the material (like the sample originally did). There could be options on the Editor script to change material import mode, but with Preview Material set by default.